dash/include/dash/halo/Halo.h

```diff
@@ -1360,14 +1594,16 @@ class HaloMemory {
    * \return Pointer to the first halo element or nullptr if the
    *         region doesn't exist
    */
-  Element_t* pos_at(region_index_t index) { return _halo_offsets[index]; }
+  Element_t* first_element_at(region_index_t index) {
```

Comment:
`first_element_at` returns a *pointer* to the first element :thinking: Why not have it return a reference?

Reply:
look at the comment above, their is the answer